great_expectations/cli/datasource.py

```diff
@@ -184,7 +184,7 @@ def add_datasource(context):
                 cli_message(
                     """
 To view the generated data documentation, open this file in a web browser:
-    <green>great_expectations/data_documentation/index.html</green>
+    <green>great_expectations/uncommitted/documentation/index.html</green>
```

Comment:
CLI's instruction was out of sync with Abe's change.

Reply:
:+1: